styles/react-select-bootstrap.css

```diff
@@ -0,0 +1,219 @@
+/**
```

Comment:
Can you include it like [https://github.com/marmelab/ng-admin/blob/master/webpack.config.js#L19](it's done in ng-admin) ?